app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/backup/RestoreFromBackupScreen.kt

```diff
@@ -275,24 +280,22 @@ private fun RestoreFromBackupContent(
         }
     }
 
-    if (state.isPasswordSheetVisible) {
-        DefaultModalSheetLayout(
-            sheetState = modalBottomSheetState,
-            enableImePadding = true,
-            wrapContent = true,
-            sheetContent = {
-                if (state.passwordSheetState is RestoreFromBackupViewModel.State.PasswordSheet.Open) {
-                    PasswordSheet(
-                        state = state.passwordSheetState,
-                        onBackClick = onBackClick,
-                        onPasswordTyped = onPasswordTyped,
-                        onPasswordRevealToggle = onPasswordRevealToggle,
-                        onPasswordSubmitted = onPasswordSubmitted
-                    )
-                }
-            },
-            onDismissRequest = onBackClick
-        )
+    if (state.passwordSheetState is PasswordSheet.Open) {
+        BottomSheetDialogWrapper(
```

Comment:
`modalBottomSheetState` and `SyncSheetState` defined above can be removed now that they're no longer used.